actors/miner/src/lib.rs

```diff
@@ -1630,11 +1662,11 @@ impl Actor {
 
             // Group declarations by deadline, and remember iteration order.
             //
-            // Vec::new() is a const expr, so this operation here is free (well, it'll add 576 bytes
-            // to the compiled binary and that'll get copied to the stack, but that should be pretty
-            // cheap.
-            const EMPTY_VEC: Vec<ExpirationExtension> = Vec::new();
-            let mut decls_by_deadline = [EMPTY_VEC; WPOST_PERIOD_DEADLINES as usize];
+            let mut decls_by_deadline =
+                Vec::with_capacity(rt.policy().wpost_period_deadlines as usize);
+            for _ in 0..rt.policy().wpost_period_deadlines {
+                decls_by_deadline.push(Vec::new());
+            }
```

Comment:
The right way to write this is `vec![Vec::new(); rt.policy().wpost_period_deadlines as usize];`.

Reply:
Well, that doesn't work because no clone... I could implement clone, but the version I pushed also works.